docs/modules/blocks/enderchests.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: enderchests
+title: Ender Chests
+---
+
+This module allow mapmakers to have better control over Ender chest behaviors in a map.
+
+The `<enderchest>` tag can contain any number of `<dropoff>` tags. Each `<dropoff>` can use a region and a filter to
+determine where and when items should be dropped from the Ender chest.
+
+<div className="alert alert--warning" style={{ marginBottom: "30px" }}>
+  Ensure that any maps with Ender chests are loaded on a server using the latest
+  version of PGM available. Ender chests in previous versions of PGM are not
+  supported and will allow players to transfer items across matches.
+</div>
+
```

Comment:
Add another note that says that ender chests cannot be placed by players during the match, they have to be in the preexisting map. There's no way to filter or allow this.